nodebuilder/node/metrics.go

```diff
@@ -33,6 +34,14 @@ func WithMetrics() error {
 		return err
 	}
 
+	buildInfoGauge, err := meter.Float64ObservableGauge(
+		"build_info",
```

Comment:
q. node-team,
`build_info` is okay or you prefer `celestia_node_build_info`

Reply:
I prefer less verbosity

Reply:
it will be prefixed with `meter` name declared on package level. In this case prefix will be `node_`. I think your naming is fine